server/buildbuddy_server/buildbuddy_server.go

```diff
@@ -142,6 +143,24 @@ func (s *BuildBuddyServer) DeleteInvocation(ctx context.Context, req *inpb.Delet
 	return &inpb.DeleteInvocationResponse{}, nil
 }
 
+func (s *BuildBuddyServer) CancelExecutions(ctx context.Context, req *inpb.CancelExecutionsRequest) (*inpb.CancelExecutionsResponse, error) {
+	err := s.authorizeInvocationWrite(ctx, req.InvocationId)
+	if err != nil {
+		return nil, err
+	}
+
+	res := s.env.GetRemoteExecutionService()
+	if res == nil {
+		return nil, status.UnimplementedError("Not Implemented")
```

Comment:
Would be good to use a more appropriate error here, something like:
status.FailedPreconditionError("remote execution not enabled")

Reply:
Oops done